src/kfs.c

```diff
@@ -19,7 +19,8 @@ struct fuse_operations kfs_ops = {
                 .link       = kfs_link,
                 .symlink    = kfs_symlink,
                 .flush      = kfs_flush,
-                .ftruncate  = kfs_ftruncate
+                .ftruncate  = kfs_ftruncate,
```

Comment:
try to do not add changes from other functionality to PR

Reply:
ftruncate was already in master